recipes/agate-stats/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{%set name = "agate-stats" %}
+{%set version = "0.4.0" %}
+{%set compress_type = "tar.gz" %}
+{%set hash_type = "sha256" %}
+{%set hash_val = "f1c68a99ffe1dfe2e71a0b9812e374bf5b3768a69ed340bfdec5951c2a3e901e" %}
+{%set build_num = "0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.{{ compress_type }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ compress_type }}
+  {{ hash_type }}: {{ hash_val }}
+
+build:
+  number: {{ build_num }}
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - six >=1.6.1
+    - agate >=1.5.0
```

Comment:
Can you try removing the `run` time dependencies from the `build` section and see how it goes?